### ISSUE TITLE
zoom further

### ISSUE THREAD

**Issue by doenutts** [lonelytb/Radar-Unknown#2]
I can only zoom in so far, how do i zoom in further?

**Comment by pubgrad** [comment]
@doenutts What happened to samuelnz vmradar? Is it deleted now?

**Comment by doenutts** [comment]
looks like it

**Comment by pubgrad** [comment]
Do you have latest zip file?

**Comment by doenutts** [comment]
i think so, if the last update was like hours ago? at like 3:42pm PST

**Comment by pubgrad** [comment]
yup it would be great if you send me download link. Did he announced he will be creating new repository or it was just sudden deletion?

**Comment by lonelytb** [comment]
`override fun scrolled`

Modify values here if you want zoom in/out further.

**Comment by pubgrad** [comment]
@doenutts nvm i think this is the same

https://github.com/ReddeR1337/VMRadar

**Comment by pubgrad** [comment]
@lonelytb in which file and line that window edge feature code is? Is it possible to add this feature to another fork which has more features then yours?

**Comment by lonelytb** [comment]
@pubgrad 
See here: #1

**Comment by doenutts** [comment]
@pubgrad my bad just saw this
